easybuild/easyconfigs/p/Pyro4/Pyro4-4.47-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = 'PythonPackage'
+
+name = 'Pyro4'
+version = '4.47'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pypi.python.org/pypi/Pyro4'
+description = """Pyro means PYthon Remote Objects. It is a library that enables you to build applications in which
+ objects can talk to eachother over the network, with minimal programming effort."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+options = {'modulename': 'Pyro4'}
+
+dependencies = [
+    ('Python', '2.7.11'),
+]
+
```

Comment:
@RvDijk please include a custom sanity check, to make sure this gets installed in the right location (i.e. **NOT** in the Python installation...)

```python
sanity_check_paths = {
    'files': [],
    'dirs': ['lib/python%(pyshortver)s/site-packages'],
}
```